src/domain/area/guidelineByAreaFilter.tsx

```diff
@@ -0,0 +1,22 @@
+import { ChangeEventHandler } from 'react-router/node_modules/@types/react'
+
+interface props {
+  guidelineNameByArea: string
+  handleSearchChange: ChangeEventHandler
+}
+const GuidelineByAreaFilter = ({ handleSearchChange, guidelineNameByArea }: props): JSX.Element => {
+  return (
+    <div className="flex flex-row justify-center w-full mt-1 ">
+      <input
+        type="search"
```

Comment:
Lets not use the type search here. It displays an x next to the input which does not align with the design.
![image](https://user-images.githubusercontent.com/14185059/136863385-4bc90a13-6c6f-4c72-b60c-56eb904f0eae.png)